pkg/partition/partition.go

```diff
@@ -109,6 +110,7 @@ func (d *Disk) addPartition() error {
 		Start: startSector,
 		End:   endSector,
 		Type:  gpt.LinuxFilesystem,
+		Label: label,
```

Comment:
@avats-dev . [This](https://github.com/openebs/node-disk-manager/blob/0baecdcae8929b6ddc6f850ca5c47eb9150af025/vendor/github.com/diskfs/go-diskfs/partition/gpt/partiton.go#L19) is the partition struct that is being used.

You need to use the [`Name`](https://github.com/openebs/node-disk-manager/blob/0baecdcae8929b6ddc6f850ca5c47eb9150af025/vendor/github.com/diskfs/go-diskfs/partition/gpt/partiton.go#L24) field here, instead of `Label`